### ISSUE TITLE
[OwnerManagement] KeyError

### ISSUE THREAD

**Issue by JeffJrShim** [Noa-DiscordBot/noa-cogs#54]
### Discord User Information

JeffJrShim#2396

### What cog is this bug report for?

OwnerManagement

#### Bug Description
You will receive a KeyError if you try to remove an owner that doesn't exist in the bot. 

#### Steps to Reproduce
1. Add an owner (can be an alt)
2. Remove the owner you added
3. Try to remove the owner you added again, and the error should appear.

#### Suggestions to fix
Use a try and except block to prevent this error from appearing.

### Please paste there error here, if any.

```python
Traceback (most recent call last):
  File "C:\Users\USER\noa\lib\site-packages\discord\ext\commands\core.py", line 85, in wrapped
    ret = await coro(*args, **kwargs)
  File "C:\Users\USER\Downloads\Noa Fukushima\data\cogs\CogManager\cogs\ownermanagement\ownermanagement.py", line 50, in remove
    self.bot.owner_ids.remove(user.id)
KeyError: 818631385392218112

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\USER\noa\lib\site-packages\discord\ext\commands\bot.py", line 939, in invoke
    await ctx.command.invoke(ctx)
  File "C:\Users\USER\noa\lib\site-packages\redbot\core\commands\commands.py", line 832, in invoke
    await super().invoke(ctx)
  File "C:\Users\USER\noa\lib\site-packages\discord\ext\commands\core.py", line 1348, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "C:\Users\USER\noa\lib\site-packages\discord\ext\commands\core.py", line 863, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "C:\Users\USER\noa\lib\site-packages\discord\ext\commands\core.py", line 94, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: KeyError: 818631385392218112
```

**Comment by Kuro-Rui** [comment]
I prefer using `if user.id in self.bot.owner_ids:` tho. You can also add that on `[p]owner add` so there are no duplicates.
Just a suggestion :)

**Comment by JeffJrShim** [comment]
> I prefer using `if user.id in self.bot.owner_ids:` tho. You can also add that on `[p]owner add` so there are no duplicates. Just a suggestion :)

Good idea, if you can PR a fix here, it would be nice. I'm quite busy atm.

**Comment by JeffJrShim** [comment]
Fixed in #56
